README.md

```diff
@@ -0,0 +1,66 @@
+# maxwells-daemon
+
+[![Build Status](https://img.shields.io/travis/thumbtack/maxwells-daemon.svg)(https://travis-ci.org/thumbtack/maxwells-daemon)
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/aws/aws-sdk-go/blog/master/LICENSE.txt)
```

Comment:
Oops, that was supposed to be THIS repo, not `aws-sdk-go`.  Also the godoc badge is useless since godoc will not generate documentation for `package main`.